docs/index.bs

```diff
@@ -3130,6 +3179,17 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Return |response|.
   </section>
 
+  <section algorithm>
+    <h3 id="skippable-fetch-handler-algorithm"><dfn>Skippable Fetch Handler</dfn></h3>
+      : Input
+      :: |serviceWorker|, a [=/service worker=]
+      : Output
+      :: a boolean
+
+      1. If |serviceWorker| has [=all fetch listeners are empty flag=] and set, then the user agent *may* return true.
```

Comment:
I think the "may" condition should sit in "Empty Handler identification", since that's the part of the process that's optional.

Reply:
Sure.  I have moved the part to "Empty Fetch Handler Identification" algorithm.